custom-metrics-stackdriver-adapter/adapter.go

```diff
@@ -18,29 +18,103 @@ package main
 
 import (
 	"flag"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	stackdriver "google.golang.org/api/monitoring/v3"
+	coreclient "k8s.io/client-go/kubernetes/typed/core/v1"
 	"os"
-	"runtime"
+	"time"
 
-	adaptersrv "github.com/GoogleCloudPlatform/k8s-stackdriver/custom-metrics-stackdriver-adapter/pkg/adapter/server"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/apiserver/pkg/util/logs"
+	"k8s.io/component-base/logs"
+	"k8s.io/klog"
 
-	// Following deps are not saved in vendor/ by godep save, so import it explicitly
-	_ "cloud.google.com/go"
-	_ "github.com/prometheus/procfs/xfs"
+	adapter "github.com/GoogleCloudPlatform/k8s-stackdriver/custom-metrics-stackdriver-adapter/pkg/adapter/provider"
+	basecmd "github.com/kubernetes-incubator/custom-metrics-apiserver/pkg/cmd"
+	"github.com/kubernetes-incubator/custom-metrics-apiserver/pkg/provider"
 )
 
+// StackdriverAdapter is an AdapterBase for Stackdriver
```

Comment:
nit: "StackdriverAdapter is an adapter for Stackdriver"

Reply:
done